fe/fe-core/src/main/java/com/starrocks/mysql/privilege/AuthUpgrader.java

```diff
@@ -270,9 +272,12 @@ protected void upgradeUserGlobalPrivileges(
                     upgradeDbPrivileges(STAR, privilege, collection, null);
                     upgradeTablePrivileges(STAR, STAR, privilege, collection, null);
                     upgradeViewPrivileges(STAR, STAR, privilege, collection, null);
+                    upgradeMaterializedViewPrivileges(STAR, STAR, privilege, collection, null);
```

Comment:
also need to support upgrade SELECT_PRIV
![image](https://user-images.githubusercontent.com/107105845/206963742-6c91fe8d-5ed3-4b53-aff3-3ab9f94363d8.png)
ps: do not forget also need to upgrade SELECT_PRIV in DbPrivTable and TablePrivTable, the pic above is only UserPrivTable